examples/inference/image_to_image.py

```diff
@@ -135,6 +147,11 @@ def __call__(
         for i, t in tqdm(enumerate(self.scheduler.timesteps[t_start:])):
             # expand the latents if we are doing classifier free guidance
             latent_model_input = torch.cat([latents] * 2) if do_classifier_free_guidance else latents
+            if isinstance(self.scheduler, LMSDiscreteScheduler):
+                sigma = self.scheduler.sigmas[t_start + i]
+                latent_model_input = latent_model_input / ((sigma**2 + 1) ** 0.5)
+                latent_model_input = latent_model_input.to(self.unet.dtype)
+                t = t.to(self.unet.dtype)
```

Comment:
For some reason `autocast` stopped working for these, so I'm adding redundant type casts. Does someone have an intuition for why that is?

Reply:
Does it work now with `autocast` or not?

Reply:
This version works with autocast, yes